cranelift/codegen/src/machinst/buffer.rs

```diff
@@ -1937,6 +1954,21 @@ pub struct MachTrap {
 pub struct MachCallSite {
     /// The offset of the call's return address, *relative to the containing section*.
     pub ret_addr: CodeOffset,
+
+    /// Range in `exception_handlers` corresponding to the exception
+    /// handlers for this callsite.
+    exception_handler_range: Range<u32>,
+}
+
+/// A call site record resulting from a compilation.
+#[derive(Clone, Debug, PartialEq)]
+pub struct FinalizedMachCallSite<'a> {
+    /// The offset of the call's return address, *relative to the containing section*.
+    pub ret_addr: CodeOffset,
+
+    /// Exception handlers at this callsite, with target offsets
+    /// *relative to the containing section*.
```

Comment:
Shouldn't these all say "relative to the containing function"? There can be multiple functions in a single section.